frontend/src/App.js

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from 'react';
+
+import {BrowserRouter, Routes, Route} from 'react-router-dom';
+
+import HomePage from './components/HomePage';
+
+import RecipePage from './components/RecipePage';
+
+import AboutPage from './components/AboutPage';
+
+function App() {
+
+    useEffect( () => console.log('Refresh'));
+
+    return (
+        <>
+        <BrowserRouter>
+        <Routes>
```

Comment:
If you leave your Navbar component between lines 17 & 18, it will render on every page so you don't have to include it in every component.